plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCStatistics.java

```diff
@@ -104,6 +104,10 @@ public long getEndTime() {
         return startTime + getTotalTime();
     }
 
+    public long getStartTime (){

```

Comment:
Could you add a whitespace here?
```suggestion
    public long getStartTime () {

```